bigquery_etl/glam/templates/clients_histogram_bucket_counts_v1.sql

```diff
@@ -95,14 +95,42 @@ unnested AS (
     normalized_histograms,
     UNNEST(histogram_aggregates) AS histogram_aggregates,
     UNNEST(aggregates) AS aggregates
+),
+-- Find information that can be used to construct the bucket range.
+-- TODO: is there a better way to incorporate the bucket information and to
+-- validate the correctness of these buckets. The Glean documentation does
+-- mention that there is a maximum recorded size, and that there are 8 buckets
+-- for each power of 2. This doesn't apply to the custom distributions
+-- (GeckoView), which would need to incorporate information from the probe info
+-- service.
```

Comment:
@emtwo what are your thoughts for bucketing glean histograms? I've been looking at the results of this query, and it looks like the number of buckets don't necessarily reflect the underlying 8*log2(n) split. The max value of the timing and memory distributions are well defined.

Reply:
This has been figured out.